### PR TITLE
fix #143 GLS FR REST incoterm

### LINKED ISSUE #143: fix incoterm for GLS FR REST
incoterm must be at the same level than "addresses" field, not inside the delivery address.

### PR DESCRIPTION
**Pull request by DylannCordel**

### CODE REVIEW

**Comment by DylannCordel** [comment]
GLS has currently some problems with UE shippings. Sometime it pass, sometime it fails with an "internal system error" from their side. If you can restart the test, it should finally success if you are lucky. :unicorn:

**Comment by DylannCordel** [comment]
rebased

**Comment by florian-dacosta** [comment]
I wonder if we should make real call to carrier webservices in tests.
It make tests really instable (not the first time I see the test fail because of gls and then success after a retry...).
Any webservice unavailability will make test fail. Maybe we should avoid this and really test some methods individually without making the call to the webservice.
Or maybe use mock or vcr...

File: roulier/carriers/gls_fr/rest/api.py [hunk 1]
```diff
@@ -27,6 +27,11 @@ def _service(self):
         schema["labelFormat"].update(
             {"type": "string", "default": "PDF", "allowed": ["PDF", "PNG"],}
         )
+        schema["incoterm"] = {
+            "type": "string",
+            # "regex": r"^\d{2}$",
+            # "allowed": ["10", "13", 18", "20", "23", "30", "40", "43", "50", "60"],
```
Comment (florian-dacosta) [inline roulier/carriers/gls_fr/rest/api.py:33]: Why leave this allowed line commented ?
These are all possible values right ?

Comment (DylannCordel) [reply roulier/carriers/gls_fr/rest/api.py:33]: Yes but those values seem to change with VAT modifications (Brexit etc.). Regex could be uncommented and allowed dropped to be sure users will refer to latests GLS doc.

**Comment by DylannCordel** [comment]
> I wonder if we should make real call to carrier webservices in tests.

I understand your worriness but in the other hand, it's the only way to be sure the implementation is not broken when carrier change it's webservice. Carriers can not get their service down for a long time. Maybe there is a way with travis to auto-schedule an other run some hours latter when some exceptions are raised ? 

We may should create a new issue for this problem.

File: roulier/carriers/gls_fr/rest/api.py (hunk 1 above)
Comment (florian-dacosta) [reply roulier/carriers/gls_fr/rest/api.py:33]: No problem for me to leave it as it is for now if it may change with time

**Pull request merged**